src/app/accounts/index.ts

```diff
@@ -15,6 +15,8 @@ export * from "./username-available"
 export * from "./get-contact-by-username"
 export * from "./update-contact-alias"
 export * from "./add-new-contact"
+export * from "./add-wallet"
```

Comment:
How are we planning to use the `src/app/wallets` dir? (I'm wondering if this file should be there or here)

Reply:
we could argue this could go to wallet as well. not sure if there is a good way to think about it.

Reply:
maybe a good counter argument is that because this method return wallet it should be under wallet.

Reply:
The problem here is that we have a circular dependency because of the `walletIds` in account. A better way would be to drop that field and have `Wallets.getWalletForAccount` just query the `wallet.accountId` (with an appropriate index).
I don't see a reason to keep the field on account - then this can clearly be part of the wallet use cases.

Reply:
> The problem here is that we have a circular dependency because of the walletIds in account. A better way would be to drop that field and have Wallets.getWalletForAccount just query the wallet.accountId (with an appropriate index).

the circular dependency has been removed. 

maybe @dolcalmi and @vindard could have another look.

note for @vindard: in many of the payments function, we're passing around `WalletId`. I think we could save a lot of back and forth to mongodb is we were passing a `Wallet`. some optimization to consider in the future.